src/AbstractOpenId.php

```diff
@@ -33,7 +33,7 @@
  *
  * @link http://openid.net/
  */
-class OpenId extends BaseClient
+abstract class AbstractOpenId extends AbstractAuthClient
```

Comment:
Let's not prefix with `Abstract`. See https://github.com/yiisoft/docs/blob/master/010-code-style.md#abstract-classes